project/AppDependencies.scala

```diff
@@ -2,13 +2,13 @@ import sbt.*
 
 object AppDependencies {
 
-  val boostrapVersion = "7.20.0"
+  val boostrapVersion = "7.21.0"
   val mongoVersion = "1.3.0"
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
```

Comment:
Can remove, not needed - `play.sbt.PlayImport.ws,`

Reply:
Done 👍